src/components/sbb-form-field/sbb-form-field.stories.tsx

```diff
@@ -202,7 +202,7 @@ const TemplateSelectWithErrorSpace = (args): JSX.Element => {
             class={args.class}
             disabled={args.disabled}
           >
-            <option>{''}</option>
+            <option value="0">{''}</option>
```

Comment:
Is the `{''}` necessary?
```suggestion
            <option value="0"></option>
```